docs/operator-manual/user-management/identity-center.md

```diff
@@ -0,0 +1,79 @@
+# Identity Center (AWS SSO)
+
+!!! note "Are you using this? Please contribute!"
+    If you're using this IdP please consider [contributing](../../developer-guide/site.md) to this document.
+
+A working Single Sign-On configuration using Identity Center (AWS SSO) has been achieved using the following method:
+
+* [SAML (with Dex)](#saml-with-dex)
+
+## SAML (with Dex)
+
+1. Create a new SAML application in Identity Center and download the certificate.
+    * ![Identity Center SAML App 1](../../assets/identity-center-1.png)
+    * ![Identity Center SAML App 2](../../assets/identity-center-2.png)
+1. Click `Assign Users` after creating the application in Identity Center, and select the users or user groups you wish to grant access to this application.
+    * ![Identity Center SAML App 3](../../assets/identity-center-3.png)
+1. Copy the Argo CD URL into the `data.url` field in the `argocd-cm` ConfigMap.
+
+<!-- markdownlint-disable MD046 -->
+```yaml
+data:
+  url: https://argocd.example.com
+```
+1. Configure Attribute mappings.
+    !!! note
+    Group attribute mapping is not officially supported in the AWS docs, however the workaround is currently working.
+    * ![Identity Center SAML App 4](../../assets/identity-center-4.png)
+    * ![Identity Center SAML App 5](../../assets/identity-center-5.png)
```

Comment:
@zeusal My apologies this is still not rendering the docs correctly.
![Screenshot 2023-10-26 at 10 23 55 AM](https://github.com/argoproj/argo-cd/assets/76892343/668ca89c-2e93-438a-a5af-d6cef740ef25)

I tested this locally and it works as expected:
```yaml
1. Configure Attribute mappings.

!!! note
    Group attribute mapping is not officially supported in the AWS docs, however the workaround is currently working.

  * ![Identity Center SAML App 4](../../assets/identity-center-4.png)
  * ![Identity Center SAML App 5](../../assets/identity-center-5.png)
  ```

Reply:
@34fathombelow I've changed it again, I think it's fine.